L01: 0000 IncGlobalCount
     0001 Function(0) -> FunctionImpl { head: 7, tail: 14, name: "foo", args: [], default_args: [], var_arg: false }
L11: 0002 PushGlobal(0) -> foo
     0003 OpFuncEval(0)
     0004 Pop
     0005 Pop
     0006 Exit
L02: 0007 Int(0) -> 0
L03: 0008 Function(1) -> FunctionImpl { head: 15, tail: 23, name: "bar", args: [], default_args: [], var_arg: false }
L08: 0009 Closure
     0010 CloseLocal(0)
L09: 0011 PushLocal(1) -> bar
     0012 OpFuncEval(0)
L10: 0013 LiftUpValue(0)
     0014 Return
L05: 0015 Function(2) -> FunctionImpl { head: 24, tail: 25, name: "_", args: [], default_args: [], var_arg: false }
     0016 OpFuncEval(0)
L06: 0017 Pop
     0018 Jump(15)
L07: 0019 PushUpValue(0)
     0020 NativeFunction(Print)
     0021 Swap
     0022 OpFuncEval(1)
L08: 0023 Return
L05: 0024 Nil
     0025 Return
Invalid 'break' statement outside of an enclosing loop
  at: line 5 (weird_loop_nesting_in_functions.cor)

5 |             (fn() { break })()
6 |                     ^^^^^
